### PR TITLE
[WIP] Update copyright notices on several files

### PR DESCRIPTION
**Pull request by tmckayus**

### CODE REVIEW

File: cloud-scripts/README.md [hunk 1]
```diff
@@ -1,3 +1,24 @@
+# SPDX-FileCopyrightText: Copyright (c) 2022 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```
Comment (ramakrishnap-nv) [inline cloud-scripts/README.md]: 2023

File: cloud-scripts/aws/README.md [hunk 2]
```diff
@@ -1,3 +1,24 @@
+# SPDX-FileCopyrightText: Copyright (c) 2022 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```
Comment (ramakrishnap-nv) [inline cloud-scripts/aws/README.md]: 2023

File: cloud-scripts/aws/aws-install-terraform.sh [hunk 3]
```diff
@@ -1,4 +1,26 @@
 #!/bin/bash
+
+# SPDX-FileCopyrightText: Copyright (c) 2022 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```
Comment (ramakrishnap-nv) [inline cloud-scripts/aws/aws-install-terraform.sh]: 2023

File: cloud-scripts/azure/README.md [hunk 4]
```diff
@@ -1,3 +1,24 @@
+# SPDX-FileCopyrightText: Copyright (c) 2022 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```
Comment (ramakrishnap-nv) [inline cloud-scripts/azure/README.md]: 2023

File: cloud-scripts/gcp/README.md [hunk 5]
```diff
@@ -1,3 +1,24 @@
+# SPDX-FileCopyrightText: Copyright (c) 2022 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```
Comment (ramakrishnap-nv) [inline cloud-scripts/gcp/README.md]: 2023

**Comment by tmckayus** [comment]
@anandhkb @rgsl888prabhu okay updated date to 2023 in those files

**Review by rgsl888prabhu — APPROVED** [review]

**Comment by tmckayus** [comment]
hmm wait, are comment markes in markdown going to render as headers?

**Comment by tmckayus** [comment]
let me try that before we merge :)

**Comment by tmckayus** [comment]
yes this is the case. never actually looked at the internal repo where this was copied from :) I'll find a way to fix, or just remove

**Pull request closed without merging**